src/read.c

```diff
@@ -5773,7 +5779,47 @@ static avifResult avifDecoderFindGainMapItem(const avifDecoder * decoder,
     return AVIF_RESULT_OK;
 }
 
-static avifResult aviDecoderCheckGainMapProperties(avifDecoder * decoder, const avifPropertyArray * gainMapProperties)
+static avifResult avifDecoderCheckAlphaProperties(avifDecoder * decoder, const avifPropertyArray * alphaProperties)
+{
+    const avifImage * image = decoder->image;
+    // The 'clap', 'irot' and 'imir' transformative properties should be applied to the alpha
+    // auxiliary image item before considering it a plane of the color image item.
+    // Otherwise, inequality with the transformative properties attached to the color image item
+    // should be treated as AVIF_RESULT_NOT_IMPLEMENTED.
+    // The latter is easier and is the behavior of libavif.
```

Comment:
This comment block (lines 5785-5789) is a little hard to understand. After reading this comment block several times, I think it will be slightly clearer if we change "Otherwise" to "Alternatively", to indicate that the first sentence and the second sentence are two options.